cli/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ _Released 02/14/2023 (PENDING)_
 
 - Fixed an issue with the Cloud project selection modal not showing the correct prompts. Fixes [#25520](https://github.com/cypress-io/cypress/issues/25520).
 - Fixed an issue in middleware where error-handling code could itself generate an error and fail to report the original issue. Fixes [#22825](https://github.com/cypress-io/cypress/issues/22825).
+- Fixed an issue where cookies were being duplicated with the same hostname, but a prepended dot. Fixed an issue where cookies may not be expiring correctly. Fixes [#25174](https://github.com/cypress-io/cypress/issues/25174), [#25205](https://github.com/cypress-io/cypress/issues/25205) and [#25495](https://github.com/cypress-io/cypress/issues/25495).
```

Comment:
seems like these should be 2 entries to me. seperate issues both related to cookies.

Reply:
I think they technically are. I can break this out into two entries

Reply:
split out in 4352ef5f3e